src/main/webapp/i18n/de/result.json

```diff
@@ -20,20 +20,24 @@
             "detail": {
                 "title": "Bewertung"
             },
-            "noResult": "Kein gewertetes Ergebnis",
+            "noResult": "Kein Ergebnis",
+            "noGradedResult": "Kein gewertetes Ergebnis",
             "resultString": "Ergebnistext",
```

Comment:
maybe you could do a nesting structure with `resultString` so you dont have to prefix all of them?